### ISSUE TITLE
Relative import path "cfb" not prefixed with / or ./ or ../

### ISSUE THREAD

**Issue by russorat** [githubocto/flat-postprocessing#4]
Hi, I recently started seeing failures in my postprocessing script: https://github.com/russorat/influxdb-data-examples/actions/runs/919922803

My code isn't the best, but it seems to be caused by https://github.com/SheetJS/sheetjs/blob/v0.16.8/types/index.d.ts#L3

I'm not using any spreadsheet parsing, so is there a way for me to just include the files i am using from https://deno.land/x/flat@0.0.10/mod.ts instead of the whole lib?

```
fetch-data
Error: Command failed: deno run -q -A --unstable ./gov/noaa/ndbc/postprocess.js ./gov/noaa/ndbc/latest-observations.txt
�[0m�[33mWarning�[0m Implicitly using latest version (1.0.0) for https://deno.land/x/url_join/mod.ts
�[0m�[1m�[31merror�[0m: Relative import path "cfb" not prefixed with / or ./ or ../ from "https://cdn.deno.land/sheetjs/versions/v0.16.8/raw/types/index.d.ts"
```

**Comment by Ambrosiani** [comment]
+1, this appeared last night. I guess some dependency has changed?

```
Error: Error: Command failed: deno run -q --allow-read --allow-write --allow-run --allow-net --allow-env --unstable postprocess.js eftervaccinet.json
error: Relative import path "ssf" not prefixed with / or ./ or ../ from "https://cdn.deno.land/sheetjs/versions/v0.16.8/raw/types/index.d.ts"
```

code: https://github.com/Ambrosiani/minnendata/blob/main/postprocess.js

**Comment by russorat** [comment]
I ~fixed~ worked around this by importing the specific files since i wasn't using sheetjs:

https://github.com/russorat/influxdb-data-examples/commit/ef5d2373e99099fa08ec3c2333a213cc8ef55044#diff-3c8aa3970d34aca51bd25c996e1dd5102e7eac927d7c62e3444751dbe1f83721L5

**Comment by Wattenberger** [comment]
This was an issue with importing the types from the sheetjs package. Fixed in [v0.0.11](https://github.com/githubocto/flat-postprocessing/releases/tag/0.0.11)
